pegasus/sites.v3/code.org/public/css/hoc-banner.scss

```diff
@@ -1,48 +1,109 @@
 @import 'color.scss', 'font.scss', 'imports/_breakpoints.scss';
 
 // Code.org HoC homepage banner
-.hoc-banner.homepage {
-  text-align: center;
-  color: $darkest_gray;
-  padding: 7em 0;
-  line-height: 1.4;
+#bars {
```

Comment:
The code in this section is referencing this: https://github.com/code-dot-org/code-dot-org/blob/848673c33968f431efe4d9c1d13ebf583db0bed9/pegasus/sites.v3/code.org/views/homepage_hero.haml#L95-L99